### PR TITLE
ENCD-4240 Allow review characterizations

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/schemas/mixins.json [hunk 1]
```diff
@@ -133,6 +133,67 @@
...
         }
     },
+    "review": { 
+        "review": {
+            "title": "Characterization review",
+            "description": "Review of the characterization results by an independent reviewer.",
+            "type": "object",
+            "additionalProperties": false,
+            "dependencies": {
+                "review_status": {
```
Comment (keenangraham) [inline src/encoded/schemas/mixins.json]: Here it is called `review_status` but in the properties it is called `status`? This works?

**Comment by keenangraham** [comment]
Also note the changelog needs to be updated.

File: src/encoded/tests/test_permissions.py [hunk 2]
```diff
@@ -140,6 +140,34 @@ def test_submitter_post_other_lab(submitter_testapp, other_lab, award, cell_free
     assert "not in user submits_for" in res.json['errors'][0]['description']
 
 
+def test_reviewer_patch_other_lab_charcaterization(submitter_testapp, testapp, submitter, attachment, construct_genetic_modification_N, other_lab, lab, award):
```
Comment (keenangraham) [inline src/encoded/tests/test_permissions.py]: Typo.

File: src/encoded/schemas/mixins.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/schemas/mixins.json]: fixed. now it is called status everywhere

File: src/encoded/tests/test_permissions.py (hunk 2 above)
Comment (gabdank) [reply src/encoded/tests/test_permissions.py]: fixed the typo

**Comment by gabdank** [comment]
@keenangraham added dependency as well as permission tests

File: buildout.cfg [hunk 3]
```diff
@@ -53,7 +53,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.24
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=idan-sno-review
```
Comment (caseylitton) [inline buildout.cfg]: flag for removal before merging

**Pull request merged**